### ISSUE TITLE
Import sqlite data

### ISSUE THREAD

**Issue by SoftTools59654** [brimdata/zui#2923]
Import sqlite data For Zui

It is currently not possible to import sqlite data

Will this feature be added in future versions?

**Comment by philrz** [comment]
@SoftTools59654: Yes, we've recognized that users would find this useful. We just need to find the time to implement it. We've already got an open issue https://github.com/brimdata/zed/issues/2844 to add the support on the Zed side which would happen first. I'll hold this issue open as a reminder that we'll need to add the support in Zui soon after that.
